Tasks/ANT/ant.ps1

```diff
@@ -87,15 +87,20 @@ $reportDirectoryName = [guid]::NewGuid()
 $reportDirectory = Join-Path $buildRootPath $reportDirectoryName
 
 $summaryFileName = "summary.xml"
+$summaryFileNameCobertura = "coverage.xml"
 $summaryFile = Join-Path $buildRootPath $reportDirectoryName 
 $summaryFile = Join-Path $summaryFile $summaryFileName
+$summaryFileCobertura = Join-Path $buildRootPath $reportDirectoryName
+$summaryFileCobertura = Join-Path $summaryFileCobertura $summaryFileNameCobertura
 # ensuring unique code coverage report task name by using guid
 $CCReportTask = "CodeCoverage_" +[guid]::NewGuid()
+# ensuring unique instrumentation task name by using guid
+$CCInstrumentTask = "CodeCoverageInstrument_" + [guid]::NewGuid()
```

Comment:
These are used as target names. We need unique target names to invoke them later. 
We call instrument target first and then the report target.